Gir.toml

```diff
@@ -318,6 +325,9 @@ status = "generate"
     name = "new"
     ignore = true
     [[object.function]]
+    pattern = "[gs]et_background_pattern"
+    ignore = true
```

Comment:
Please add comment, why it ignored

Reply:
Especially if it is not manually implemented either

Reply:
Deprecated. I'll add the comment.

Reply:
It deprecated in 3.22 https://developer.gnome.org/gdk3/stable/gdk3-Windows.html#gdk-window-get-background-pattern, so it can be used even in max available on CI 3.18.
So this not good reason to ignore.

Reply:
Damn. You're hard in business. :stuck_out_tongue: